src/stories/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,51 @@
+import './styles.scss';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Component for sidebar element.
+ *
+ * @component
+ * @param {string} modifierClasses Class modifiers of the component.
+ * @param {html} sidebarContent Sidebar content of the component.
+ * @param {html} mainContent Main content of the component.
```

Comment:
I recently found out the correct parameter type for this is `node`.

Reply:
Thank you!